index.html

```diff
@@ -140,36 +46,156 @@ <h4 class="connect">Let's Connect</h4>
                 </li>
                 <li>
-                    <img src="assets/css-icon.png" alt="Css Icon">
-                    <p>CSS</p>
+                    <a href="https://github.com/kelvinbush">
+                        <img src="assets/cool-icon.svg" alt="Cool Icon"></a>
                 </li>
             </ul>
-        </li>
-        <li class="language_container closed">
-            <p>Frameworks</p>
-            <img src="assets/arrow_right.svg" alt="Down Arrow">
-        </li>
-        <li class="language_container closed">
-            <p>Skills</p>
-            <img src="assets/arrow_right.svg" alt="Down Arrow">
-        </li>
-    </ul>
-</section>
+            <button id="btn-resume" type="button">Get my resume</button>
+        </div>
+        <ul>
+            <li class="language_container open">
+                <p>Languages</p>
+                <img src="assets/arrow_down.svg" alt="Down Arrow">
+                <ul class="about_languages">
+                    <li>
+                        <img src="assets/javascript.png" alt="Javascript Logo">
+                        <p>Javascript</p>
+                    </li>
+                    <li>
+                        <img src="assets/html-icon.png" alt="Html Icon">
+                        <p>HTML</p>
+                    </li>
+                    <li>
+                        <img src="assets/css-icon.png" alt="Css Icon">
+                        <p>CSS</p>
+                    </li>
+                </ul>
+            </li>
+            <li class="language_container closed">
+                <p>Frameworks</p>
+                <img src="assets/arrow_right.svg" alt="Down Arrow">
+            </li>
+            <li class="language_container closed">
+                <p>Skills</p>
+                <img src="assets/arrow_right.svg" alt="Down Arrow">
+            </li>
+        </ul>
+    </section>
+    <section class="contact">
+        <div class="container">
+            <h2>Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built
+                or a project that needs coding. I’d love to help with it </p>
+            <form method="post" action="https://formspree.io/f/xqknloew">
+                <label>
+                    <input type="text" name="name" required maxlength="30" placeholder="Your name">
+                </label>
+                <label>
+                    <input type="email" name="email" required placeholder="Your email">
+                </label>
+                <label id="text-area">
+                <textarea name="message" required maxlength="500"
+                          placeholder="Write your message here"></textarea>
+                </label>
+                <label id="submit-button">
+                    <button id="submit-input" type="submit">Get In Touch</button>
+                </label>
+            </form>
+        </div>
+    </section>
```

Comment:
- [ ] [Optional] It would be better semantically if you add the `contact me` section inside a `<footer>` tag instead of `<section>` tag because it's the last section on the page and there are no elements under it to be the footer.
- [ ] [Optional] It would be better if you remove `<label>` tags and put `input` fields directly inside the `<form>` tag because there is no title (label) above each input field.